nemo/lightning/io/artifact/file.py

```diff
@@ -19,8 +20,7 @@ class FileArtifact(Artifact[str]):
     def dump(self, value: str, absolute_dir: Path, relative_dir: Path) -> str:
         if not pathize(value).exists():
             # This is Artifact is just a string.
-            self.skip = True
-            return value
+            return fdl.Config(FileArtifact, attr=value, skip=True)
```

Comment:
Why is `skip` `True` by default here? Shouldn't it be equal to `self.skip`?

Reply:
it's true inside the if's body, but it does not matter, [later ](https://github.com/NVIDIA/NeMo/pull/10936/files#diff-726b301346c45eb4d0766d79e06ae22cd71ea1740eee2b1fbc0aee92c8045f61R616) the code checks if it's a fdl.Config